powerline/config_files/config.json

```diff
@@ -46,7 +46,9 @@
 		},
 		"wm": {
 			"colorscheme": "default",
-			"theme": "default"
+			"theme": "default",
+      "local_themes": {
+      }
```

Comment:
Also this change without actually adding any local themes is not needed, revert.